docs/install.rst

```diff
@@ -77,11 +77,11 @@ To ensure that Horovod is built with MXNet CPU support enabled:
 
     $ HOROVOD_WITH_MXNET=1 pip install horovod[mxnet]
 
-To ensure that Horovod is built with MXNet GPU support enabled for CUDA 10.2:
+Some MXNet versions do not work with Horovod:
 
-.. code-block:: bash
-
-    $ HOROVOD_WITH_MXNET=1 pip install horovod[mxnet-cu102]
```

Comment:
If removing this, can we replace with another step for how to install the correct version of MXNet?

Reply:
@tgaddair as I understood you, we do not want to go into this detail, right?

Reply:
I think we can leave the example install command, if we also add these version restrictions to the `mxnet-cu102` extra.